### ISSUE TITLE
Multi-view image reconstruction failure

### ISSUE THREAD

**Issue by surpoloyang** [Inception3D/TTT3R#4]
I used a set of multi-view images of plants, hoping to reconstruct a 3D model of the entire plant.
I used the default parameters of the demo, loaded the provided pre-trained weights, but the resulting glb file only shows the camera position.

<img width="800" height="392" alt="Image" src="https://github.com/user-attachments/assets/049865e5-ae4b-427c-ac06-073b97d7e988" />

**Comment by rover-xingyu** [comment]
Hi @surpoloyang, the camera poses look good. Maybe the point clouds are filtered out with [conf_other_tosave > 1](https://github.com/Inception3D/TTT3R/blob/cc6142bd9c15df57817a5c7a4797c163e46fa221/demo.py#L366C105-L366C126)

We recommend visualizing the results in Viser. There you can adjust the “Visibility Threshold” slider to pick a confidence level that suits you. Please give it a try, and let me know how it goes—I’m looking forward to seeing your results!

**Comment by surpoloyang** [comment]
Not as good as expected, as shown below:

<img width="2560" height="1229" alt="Image" src="https://github.com/user-attachments/assets/64e30c51-aae6-40e4-bf48-dde053649175" />

**Comment by surpoloyang** [comment]
<img width="1577" height="969" alt="Image" src="https://github.com/user-attachments/assets/ff71d8f4-a56e-4e76-aa77-82de46ca89c5" />

**Comment by rover-xingyu** [comment]
Oh, this visualization config is for long sequences (e.g., >1000 frames).

For short sequences, adjust these visualization settings in Viser:
- Train Step → 102
- Show pointclouds → 3D (All Frames)
- Downsample Factor → 1

Try again with these settings!

**Comment by surpoloyang** [comment]
You can see the point cloud of the object, but the effect is off, and the point cloud didn't match. The images from training steps 1->102 are as follows:

<img width="1084" height="975" alt="Image" src="https://github.com/user-attachments/assets/cdbed0ab-a192-4248-929d-11a53a6c3e25" />

<img width="1385" height="966" alt="Image" src="https://github.com/user-attachments/assets/d70a6549-0313-4958-a2b0-23fb78cd4810" />

<img width="1443" height="999" alt="Image" src="https://github.com/user-attachments/assets/de660104-6ca9-499d-8e56-9e5615a77020" />

**Comment by rover-xingyu** [comment]
I see. The black background is the devil. Unfortunately, TTT3R can't handle this case yet. 
Please try with the original images instead of the masked ones if available.

**Comment by surpoloyang** [comment]
Okay, thank you! Do not use a mask mean separating the main plant from the image and saving the main plant as a new image, then inferring?

**Comment by rover-xingyu** [comment]
Yes, CUT3R is trained on images with natural backgrounds, so both CUT3R and TTT3R cannot handle masked images with black/white backgrounds.

**Comment by surpoloyang** [comment]
Thank you for your replies!
